crates/voicevox_core/src/internal.rs

```diff
@@ -148,7 +146,7 @@ impl Internal {
         );
 
         self.synthesis_engine
-            .synthesis_wave_format(&audio_query, speaker_id, false)
+            .synthesis_wave_format(&audio_query, speaker_id, true)
```

Comment:
ENGINEに合わせると、疑問文っぽくするかを外部から設定する形にしたいかもです！
こんな感じのTODOコメント追加とかいかがでしょう。
```suggestion
            .synthesis_wave_format(&audio_query, speaker_id, true) // TODO: 疑問文化を設定可能にする
```
（linter的にこの書き方で良いのかは自信がないです･･･）

Reply:
`voicevox_tts` に増やすのはちょっと考えた方がいいかもしれません。
そう頻繁に引数が増えることはないと思いますが、initializeのように引数がどんどん増えていってしまう可能性もありますし、慎重にやったほうがよさそうに感じました。また `voicevox_tts` の引数が増えてしまうとユーザーサイドに混乱を与える可能性もあります。
アプローチ方法として `voicevox_tts` はそのままにしてシンプルに使用できるようにし、 `voicevox_tts_with_options` といった関数を作って詳細に設定したい人はそっちを使用するように誘導するといった方法もあるかなと思いました

Reply:
なるほど、了解です！
この PR で仕様変更して良いのであれば、ちょっと `voicevox_tts` 関数の引数を足してみます。

Reply:
`voicevox_tts` 関数の引数に以下のように `enable_interrogative_upspeak` を追加してみました：

```c++
VoicevoxResultCode voicevox_tts(const char *text,
                                int64_t speaker_id,
                                int *output_binary_size,
                                uint8_t **output_wav,
                                bool enable_interrogative_upspeak);
```

Reply:
もしかしたら @qwerty2501 さんのコメントと行き違いになってしまったかも？ 👀

たしかにoptionパラメータは増えていきそうな気がしています。
あとたぶん将来的にaccent_phraseかaudio_queryから合成する関数が生えたとき、同じ引数が並ぶことになりそう。（エンジンのAPIがそうなってます）

個人的には、まあAPI側は半年くらい引数増えてないからそう高頻度に増えないと見ていったん引数を追加する運用でもいいのかな～とちょっと思いました。
か、まあまだリリース前なので、一旦PRマージ後にoptions化を検討するのも良いかも。

Reply:
> もしかしたら @qwerty2501 さんのコメントと行き違いになってしまったかも？ 👀

はい、行き違いになってしまいました（汗）

私も実際 @qwerty2501 さんと同じくパラメータを安直に増やして良いものか懸念を抱いていたので、一応あらかじめ https://github.com/VOICEVOX/voicevox_core/pull/192#discussion_r924776135 で質問した形です。ちょっと判断が難しいですね……（悩んだので、試しに変更を commit してみて議論につなげられたらという意図もありました）。

悩んだのですが、よく考えるとどんな API のデザインにしたいかということはそもそも Rust 対応の範疇ではないと思ったので、元の形に戻して TODO コメントを入れる形にしたいと思います……！

Reply:
たしかに、API設計に関してはいろいろ完了したあとの方が良さそうですね！
ありがとうございます！！